addons/event_sms/models/event_mail.py

```diff
@@ -56,9 +56,10 @@ def execute(self):
                         template=scheduler.template_ref,
                         mass_keep_log=True
                     )
+                    attendees_in_draft = len(scheduler.event_id.registration_ids.filtered(lambda r: r.state == "draft"))
                     scheduler.update({
                         'mail_done': True,
-                        'mail_count_done': scheduler.event_id.seats_reserved + scheduler.event_id.seats_used,
+                        'mail_count_done': scheduler.event_id.seats_reserved + scheduler.event_id.seats_used + attendees_in_draft
```

Comment:
Don't forget this one for consistency :p

same for whatsapp PR

Reply:
yeah, that would be usefull